app/views/measure_conditions/_measure_condition_code_default.html.erb

```diff
@@ -13,7 +13,7 @@
     <tr class="govuk-table__row">
       <td class="govuk-table__cell numerical"><%= condition.document_code %></td>
       <td class="govuk-table__cell">
-        <%= condition.requirement.to_s.html_safe %>
+        <%= condition.certificate_description.presence || condition.requirement.to_s.html_safe %>
```

Comment:
does the description of the ticket
"Show certificate description instead of requirement"
imply that `condition.requirement` has priority on the `certificate_description`?